google/cloud/spanner/samples/samples.cc

```diff
@@ -1354,13 +1363,14 @@ void QueryWithStringParameter(google::cloud::spanner::Client client) {
 //! [END spanner_query_with_string_parameter]
 
 //! [START spanner_query_with_timestamp_parameter]
-void QueryWithTimestampParameter(google::cloud::spanner::Client client) {
+void QueryWithTimestampParameter(
+    google::cloud::spanner::Client client,
+    google::cloud::spanner::Timestamp last_update_time) {
   namespace spanner = ::google::cloud::spanner;
-  auto example_timestamp = spanner::MakeTimestamp(absl::Now()).value();
   spanner::SqlStatement select(
       "SELECT VenueId, VenueName, LastUpdateTime FROM Venues"
       " WHERE LastUpdateTime <= @last_update_time",
-      {{"last_update_time", spanner::Value(example_timestamp)}});
+      {{"last_update_time", spanner::Value(last_update_time)}});
```

Comment:
This example may be hard to grok when reading it at cloud.google.com, the context of where `last_update_time` comes from is lost. Consider putting the body of the query inline the example.  Alternatively, one can reopen a region (in this case the `[START spanner_query_with_timestamp_parameter]`), so you could refactor this example to call a version of `DatabaseNow()` which is included in the example.  I say I version because I would pass an already built `spanner::Client` to it.

Reply:
I think losing the particular value of `last_update_time` from the snippet is the goal.  The value is arbitrary.  The snippet is only about how to interpolate it into a query.  Yes?